BASH/downloader.sh

```diff
@@ -102,10 +115,12 @@ do
 	path=$(pwd)
 	echo -e "Your downloaded file is in this path:\n" $path
 	cd ..
-        if [ "$x" != "" ]; then
-        nautilus $path
+	if [ "$x" != "" ]; then
+		nautilus $path
 	else
-	gnome-open chap$chapno.pdf	
- 	rm -rf $chap
-        fi
+		if [ -n "$openWith" ]; then
+			$openWith chap$chapno.pdf
+		fi
+		rm -rf $chap
+	fi
```

Comment:
Indentation

Reply:
I don't see a problem. Can you elaborate a bit more mentioning the line number.

Reply:
It just looked wrong in the git diff view